package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "calculator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
- [ ] [REQUIRED] [big.js](https://github.com/MikeMcl/big.js/) added to package.json. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/big.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
 - [ ] [REQUIRED] [prop-types](https://www.npmjs.com/package/prop-types) added to package.json for [typechecking props](https://reactjs.org/docs/typechecking-with-proptypes.html) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/section/propTypes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))